addon/components/universal-export/component.js

```diff
@@ -29,6 +31,16 @@ export default Component.extend({
 
   currentWindow: 'file',
   filters: [],
+  hideInfluencerNetwork: false,
+
+  hasDisabledInfluencerNetworkModal: computed(function() {
+    const cookieValue = document.cookie
+      .split('; ')
+      .find(row => row.startsWith(INFLUENCER_NETWORK_MODAL_COOKIE))
+      .split('=')[1];
```

Comment:
there is not a better way to manipulate cookie ?

Reply:
Hmm that's the shortest way I know, cookies are basically one big string so there's not so many options
We can install ember-cookies but it's not worth it IMO